.github/workflows/ci.yml

```diff
@@ -144,3 +144,70 @@ jobs:
         set PYTHONPATH=%PYTHONPATH%;%GITHUB_WORKSPACE%/bin/%KRATOS_BUILD_TYPE%
         set PATH=%PATH%;%GITHUB_WORKSPACE%/bin/%KRATOS_BUILD_TYPE%/libs
         python kratos/python_scripts/run_tests.py -l small -c python
+
+
+
+  windows-core-full-debug:
+    runs-on: windows-latest
+    env:
+      KRATOS_BUILD_TYPE: FullDebug
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.6'
+    - name: Cache Build
+      id: cache-build
+      uses: actions/cache@v1
+      with:
+        path: build
+        key: ${{ runner.os }}-build-${{ github.sha }}
+        restore-keys: ${{ runner.os }}-build-
+    - name: Build
+      shell: cmd
+      run: |
+        call "%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64 || goto :error
+
+        set CC=cl.exe
+        set CXX=cl.exe
+
+        set KRATOS_SOURCE=%cd%
+        set KRATOS_BUILD=%cd%\build
+
+        del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\cmake_install.cmake"
+        del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\CMakeCache.txt"
+        del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\CMakeFiles"
+
+        cmake                                                ^
+          -G"Visual Studio 16 2019"                          ^
+          -H"%KRATOS_SOURCE%"                                ^
+          -B"%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%"             ^
+          -DBOOST_ROOT="%BOOST_ROOT_1_72_0%"                 ^
+          -DINSTALL_RUNKRATOS=OFF                            ^
+          -DCMAKE_CXX_FLAGS="/Od                             ^
```

Comment:
@roigcarlo what to do here? Remove?

Reply:
You can leave it, fulldebug also adds that. It was on the regular one so it does not enable optimization which make is took much more time to compiple